config/default/manager_default_images.yaml

```diff
@@ -12,8 +12,8 @@ spec:
       - name: manager
         env:
         - name: INFRA_CLIENT_IMAGE_URL_DEFAULT
-          value: quay.io/tripleozedcentos9/openstack-tripleoclient:current-tripleo
+          value: quay.io/podified-antelopecentos9/openstack-openstackclient:current-podified
```

Comment:
Oops. Sorry this was wrong. I've created a PR to fix this (and the other wrong names)